src/components/Subscribe/_subscribe.scss

```diff
@@ -4,34 +4,57 @@
 @import '#scss/typography';
 
 .#{$px}-subscribe {
-  max-width: 30rem;
+  max-width: 600px;
   width: 100%;
 
+  .#{$px}-input {
+    &__label {
+      color: $pure-black;
+      font-size: 0.75rem;
+      text-transform: none;
+    }
+
+    &__validation {
+      padding-left: 0;
+    }
+  }
+
   &__title {
     @include text($heading3);
 
     margin-bottom: $spacing-xsm;
-    text-align: center;
   }
 
   &__blurb {
     @include text($body2);
 
-    text-align: center;
-  }
-
-  @include media($size-sm) {
-    &__title,
-    &__blurb {
-      text-align: left;
-    }
+    color: $pure-black;
   }
 
   &__input__wrapper {
     margin-top: $spacing-md;
   }
 
   &__button {
-    margin: $spacing-md 0 $spacing-md;
+    height: 36px;
```

Comment:
I am not a fan of setting explicit heights on things. The padding, font size, and line-height should equal the height of the button. Adding up the values they have in [this figma](https://www.figma.com/design/koazePmhaPVdUszim1NGTP/Navigation?node-id=2210-1384&m=dev) does not add up to what is listed.